src/test/ts/part2/Exercise3OptionTest.ts

```diff
@@ -1,5 +1,7 @@
 import { describe, it } from '@ephox/bedrock-client';
-import { assert } from 'chai';
+import { Optional } from '@ephox/katamari';
+import { createElement } from '@ephox/sugar/lib/main/ts/ephox/sugar/api/node/SugarShadowDom';
```

Comment:
Another case where we should be imported the named export instead:
```suggestion
import { SugarShadowDom } from '@ephox/sugar';
```